### PR TITLE
GPIB-462

### PR DESCRIPTION
**Pull request by xiangzhe0812**
add QRCode on the verify email page

### CODE REVIEW

**Review by rellfy — DISMISSED** [review]

File: src/components/VerificationTracker/VerifyEmail.js [hunk 1]
```diff
@@ -19,24 +20,37 @@ const VerifyEmail = () => {
...
+            onClick={onResendClick}
+            icon={hasResent && !error && "checkmark-circle"}
+          />
+        </div>
+        <div>
+          <p className="ml-3">
+            <b>Use Idem App to Verfiy</b>
+          </p>
+          <div className="p-2">
+            <QRCode value={idemUrl ? idemUrl : null} size="200" />
```
Comment (ahinchliff) [inline src/components/VerificationTracker/VerifyEmail.js]: Does it make sense to remove the QRCode component from the DOM if there isnt a idemURL? What does an empty QRCode look like?

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
We will need this behind a feature toggle so we can turn on when IDEM goes live

**Comment by xiangzhe0812** [comment]
just make PR draft, can reopen when idem go alive

**Review by bitcoinbrisbane — DISMISSED** [review]
LGTM

File: src/components/VerificationTracker/VerifyEmail.js [hunk 2]
```diff
@@ -19,24 +20,39 @@ const VerifyEmail = () => {
...
+            submitText="Resend verification email"
+            block={false}
+            onClick={onResendClick}
+            icon={hasResent && !error && "checkmark-circle"}
+          />
+        </div>
+        {idemUrl && (
+          <div>
+            <p className="ml-3">
+              <b>Use Idem App to Verfiy</b>
```
Comment (RalphLavelle) [inline src/components/VerificationTracker/VerifyEmail.js]: "Verify"

File: src/components/VerificationTracker/VerifyEmail.js [hunk 3]
```diff
@@ -19,24 +20,39 @@ const VerifyEmail = () => {
...
-        icon={hasResent && !error && "checkmark-circle"}
-      />
+      <div className="d-flex justify-content-between align-items-center">
+        <div>
+          <p>
+            <b>Verify your email to continue</b>
+          </p>
+          <ErrorMessage error={error} />
+          <p>
+            We have sent you a link to verify your email. <br></br>
```
Comment (RalphLavelle) [inline src/components/VerificationTracker/VerifyEmail.js]: Why the two "<brs>"? Just make it a new paragraph

File: src/components/VerificationTracker/VerifyEmail.js [hunk 4]
```diff
@@ -19,24 +20,39 @@ const VerifyEmail = () => {
...
+            icon={hasResent && !error && "checkmark-circle"}
+          />
+        </div>
+        {idemUrl && (
+          <div>
+            <p className="ml-3">
+              <b>Use Idem App to Verfiy</b>
+            </p>
+            <div className="p-2">
+              <QRCode value={idemUrl ? idemUrl : null} size="200" />
```
Comment (RalphLavelle) [inline src/components/VerificationTracker/VerifyEmail.js]: This seems pointless as it doens't do anything with it if it's null.
`<QRCode value={idemUrl} size="200" />`

File: src/components/VerificationTracker/VerifyEmail.js (hunk 2 above)
Comment (bitcoinbrisbane) [reply src/components/VerificationTracker/VerifyEmail.js]: Seems to be fixed.

**Comment by bitcoinbrisbane** [comment]
Will need the to add `REACT_APP_IDEM_URL` in the .env example too.

File: src/components/VerificationTracker/VerifyEmail.js (hunk 4 above)
Comment (bitcoinbrisbane) [reply src/components/VerificationTracker/VerifyEmail.js]: Seems to be fixed.

**Review by rellfy — APPROVED** [review]

**Review by bitcoinbrisbane — APPROVED** [review]

**Pull request merged**